core/src/components/item/test/highlight/item.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('item: highlight', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/item/test/highlight`);
+
+    await page.setIonViewport({ resizeViewportWidth: true });
```

Comment:
Question for this configuration used in both tests - were you observing horizontal clipping of the screenshot viewport? 

Based on the screenshot captures, I think this can likely be removed if not.

```ts
await page.setIonViewport();
```

The `resizeViewportWidth` option was introduced for 1 item test we have, where the rendered contents are larger than the device width (some legacy test that renders on like a landscape tablet viewport width).

Reply:
Oh no I just copied from another test, but also I saw that `setIonViewport` was used to take a full height screenshot so I thought that would make it work. I don't think it did though ha.

Reply:
The `resizeViewportWidth` option can also likely be removed from the `fill` test. `setIonViewport` by itself will handle resizing the height 👍